docs/developers_guide/Testing.md

```diff
@@ -68,11 +68,16 @@ You can inject list of your own or mocked components with `.withExtraComponents`
 Each component should match `ComponentDefinition`. It means that ComponentDefinition passed via `.withExtraComponents` overrides the one which could be already defined in modelData for given name. Example:
 ```scala
 import pl.touk.nussknacker.engine.flink.util.test.FlinkTestScenarioRunner._
+import pl.touk.nussknacker.engine.util.functions._
+
 val stubbedGetCustomerOpenApiService: SwaggerEnricher = new SwaggerEnricher(Some(new URL(rootUrl(port))), services.head, Map.empty, stubbedBackedProvider)
 val mockComponents = List(ComponentDefinition("getCustomer", stubbedGetCustomerOpenApiService))
+val globalVariables = Map("DATE", date)
+
 val testScenarioRunner = TestScenarioRunner
       .flinkBased(resolvedConfig, flinkMiniCluster)
       .withExtraComponents(mockComponents)
+      .withGlobalVariables(globalVariables)
```

Comment:
Please don't mix both things. We have to extract another section for global variables